ecs-logs-collector.sh

```diff
@@ -385,7 +385,7 @@ get_ecs_agent_logs() {
 
   mkdir -p "$dstdir"
 
-  cp -f /var/log/ecs/* "$dstdir"/
+  cp -Rf /var/log/ecs/* "$dstdir"/
```

Comment:
this could get problematic, if customer has some other set of logs/other confidential info going to this location - might be preferable if we obtain the logs from exec dir

Reply:
I was thinking bout this, and actually I am not sure that we even should copy the exec logs themselves. Unlike ECS Agent logs, Docker logs, etc. I'm am guessing Exec logs could have confidential data (AFAIK, ssm logs the commands executed, so some sensitive data might be logged?)

Reply:
Also, by the same token. Shouldn't we be changing the current command from `cp -f /var/log/ecs/* "$dstdir"/` to  `cp -f /var/log/ecs/ecs-* "$dstdir"/` in case there's something confidential being logged to ` /var/log/ecs/confidential-stuff.log`?

Reply:
that is true, the folder has more than just agent logs, it also contains the audit logs and ecs volume plugin logs. we could change this to collect those alone. we might need ssm logs for debugging ssm agent failures etc., but if we feel its logging more customer data, we can skip its collection and defer it to the customer (also, could check what SSM team is doing today)